src/client/features/view/components/sidebar/menus/graphInfoMenu.js

```diff
@@ -40,24 +16,28 @@ class GraphInfoMenu extends React.Component {
 
     const datasourceLink = this.getDatasourceLink(props.datasource);
 
+    const noInfoMessage = h('span', [
+      h('p', 'No additional information was found for this network!'),
+      h('p', 'Additional information about the network is normally found here, but we couldn\'t find any for this one.')
+    ]);
+
     const comments = props.comments.map(comment => {
-      return h('div', [
-        comment.replace(/<p>/g, ' '),
-        h('br'),
-        h('br')
-      ]);
+      return h('div', [h('p', comment.replace(/<p>/g, ' '))]);
     });
 
     return (
       h('div', [
         h('h1', props.name),
         h('h4', [
           'Sourced from ',
-          h('a', { href: datasourceLink, target: '_blank'}, props.datasource)
+          h('a', { href: datasourceLink, target: '_blank' }, props.datasource)
         ]),
-        h('div', [
-          h('h2', 'Additional Information')
-        ].concat(comments))
+        ...(comments.length ?
```

Comment:
Minor issue but could you move this chunk of conditional logic outside of the rendering statement?  It would be easier to read. 

const yourVariableHere = ...(comments.length ? ...) : [noInfoMessage]

i.e assign this statement to a variable and then replace this to:
h('div', yourVariableHere)

Reply:
Makes sense. Just fixed it.